packages/core/stories/patterns/keyboard-shortcuts/keyboard-shortcuts.stories.tsx

```diff
@@ -0,0 +1,325 @@
+      alert("Duplicate ticket triggered");
+    },
+    { enabled: shortcutsEnabled },
+  );
+  useHotkeys(
+    "meta+b",
+    (e) => {
+      shortcutsEnabled && alert("Set direction to buy triggered");
+    },
+    { enabled: shortcutsEnabled },
+    [shortcutsEnabled],
+  );
+  useHotkeys(
+    "meta+s",
+    (e) => {
+      e.preventDefault();
+      alert("Set direction to sell triggered");
+    },
+    { enabled: shortcutsEnabled },
+  );
+  useHotkeys(
+    "meta+end",
+    (e) => {
+      e.preventDefault();
+      alert("Bottom of list triggered");
+    },
+    { enabled: shortcutsEnabled },
+  );
+  useHotkeys(
+    "meta+home",
+    (e) => {
+      e.preventDefault();
+      alert("Top of list triggered");
+    },
+    { enabled: shortcutsEnabled },
+  );
+  useHotkeys(
+    "meta+u",
+    (e) => {
+      e.preventDefault();
+      alert("Test shortcut triggered");
+    },
+    { enabled: shortcutsEnabled },
+  );
+  useHotkeys(
+    "meta+y",
+    (e) => {
+      e.preventDefault();
+      alert("Test shortcut triggered");
+    },
+    { enabled: shortcutsEnabled },
+  );
+  useHotkeys(
+    "meta+shift+k",
+    (event) => {
+      event.preventDefault();
+      setFilter("");
+      setOpen(true);
+    },
+    { enabled: shortcutsEnabled },
+  );
+
+  const filteredShortcuts: Shortcut[] = shortcutList.filter((s) =>
+    s.label.toLowerCase().includes(filter.trim().toLowerCase()),
+  );
+
+  const handleDialogOpen = (): void => {
+    setFilter("");
+    setOpen(true);
+  };
+  const handleDialogChange = (value: boolean): void => {
+    setOpen(value);
+    if (value) setFilter("");
+  };
+  const handleSwitchChange = (event: ChangeEvent<HTMLInputElement>): void =>
+    setShortcutsEnabled(event.target.checked);
+  const handleFilterChange = (event: ChangeEvent<HTMLInputElement>): void =>
+    setFilter(event.target.value);
+
+  return (
+    <HotkeysProvider>
+      <StackLayout gap={1}>
+        <Button data-testid="dialog-button" onClick={handleDialogOpen}>
+          Keyboard shortcuts panel
+        </Button>
+        <FlexLayout align="center" gap={1}>
+          <Text>hit </Text>
+          <FlexLayout align="center" gap={0}>
+            <Kbd>meta</Kbd>+<Kbd>shift</Kbd>+<Kbd>K</Kbd>
+          </FlexLayout>
+          <Text>to open the keyboard shortcuts panel </Text>
+        </FlexLayout>
+      </StackLayout>
+      <Dialog
+        open={open}
+        onOpenChange={handleDialogChange}
+        id="keyboard-shortcuts-dialog"
+        size="medium"
+        className="keyboardShortcuts-dialog"
+      >
+        <DialogContent>
+          <StackLayout gap={3}>
+            <FlexLayout gap={1}>
+              <Switch
+                checked={shortcutsEnabled}
+                onChange={handleSwitchChange}
+              />
+              <FlexLayout className="keyboardShortcuts-description">
+                <Text>Turn on keyboard shortcuts</Text>
+              </FlexLayout>
+            </FlexLayout>
```

Comment:
```suggestion
              <Switch
                checked={shortcutsEnabled}
                onChange={handleSwitchChange}
                label="Turn on keyboard shortcuts"
              />
```
Not sure if there was a specific reason we split the label out into a separate <Text> block, but I think this would be a bit cleaner + more consistent with the Switch pattern if we use the built-in label prop. Also, doing it this way ensures the Switch has an accessible name. Thoughts?